ddl.sql

```diff
@@ -47,635 +50,815 @@ CREATE TABLE Email_pracownik(
 	Data_do DATE
 );
 
-CREATE TABLE Stanowisko(
+CREATE TABLE Stanowisko
+	(
 	ID_stanowisko INT PRIMARY KEY IDENTITY(1,1) NOT NULL,
 	Nazwa_stanowiska VARCHAR(50) NOT NULL
 );
 
-CREATE TABLE Dzial(
+CREATE TABLE Dzial
+	(
 	ID_dzial INT PRIMARY KEY IDENTITY(1,1) NOT NULL,
 	Nazwa_dzialu VARCHAR(50) NOT NULL
 );
 
-CREATE TABLE Rodzaj_kontrola(
-ID_rodzaj_kontrola INT PRIMARY KEY IDENTITY(1,1) NOT NULL,
-Rodzaj_kontrola NVARCHAR(50) NOT NULL,
-Procedura NVARCHAR(250) NOT NULL
+CREATE TABLE Rodzaj_kontrola
+	(
+	ID_rodzaj_kontrola INT PRIMARY KEY IDENTITY(1,1) NOT NULL,
+	Rodzaj_kontrola NVARCHAR(50) NOT NULL,
+	Procedura NVARCHAR(250) NOT NULL
 );
 
-CREATE TABLE Rezultat_kontrola(
-ID_rezultat INT PRIMARY KEY IDENTITY(1,1) NOT NULL,
-Wynik NVARCHAR(50) NOT NULL,
-Uwagi NVARCHAR(250)
+CREATE TABLE Rezultat_kontrola
+	(
+	ID_rezultat INT PRIMARY KEY IDENTITY(1,1) NOT NULL,
+	Wynik NVARCHAR(50) NOT NULL,
```

Comment:
to będzie wynikać już z dokonanych pomiarów i tolerancji, nie potrzebujemy tabeli